package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "weather-app",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack",
+    "watch": "webpack --watch"
```

Comment:
The `mode` option has not been set, webpack will fallback to `production` for this value.
Set the `mode` option to `development` or `production` to enable defaults for each environment.
You can also set it to `none` to disable any default behavior. [Learn more](https://webpack.js.org/configuration/mode/)